src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Logging;
+using Dynamo.Models;
+using Dynamo.PythonMigration.Properties;
+using Dynamo.UI.Prompts;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.Extensions;
+using PythonNodeModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media;
+
+namespace Dynamo.PythonMigration
+{
+    public class PythonMigrationViewExtension : IViewExtension
+    {
+        private const string EXTENSION_NAME = "Workspace References";
+        private const string EXTENSION_GUID = "1f8146d0-58b1-4b3c-82b7-34a3fab5ac5d";
+        private const string NOTIFICATION_TITLE = "IronPythonNotification";
+
+        private ViewLoadedParams LoadedParams { get; set; }
+        private DynamoViewModel DynamoViewModel { get; set; }
+        private NotificationMessage IronPythonNotification { get; set; }
+
+        /// <summary>
+        /// Extension GUID
+        /// </summary>
+        public string UniqueId { get { return EXTENSION_GUID; } }
+
+        /// <summary>
+        /// Extension Name
+        /// </summary>
+        public string Name { get { return EXTENSION_NAME; } }
+
+        public void Shutdown()
+        {
+        }
+
+        public void Startup(ViewStartupParams p)
+        {
+        }
+
+        public void Dispose()
+        {
+            EventUnsubscriber();
```

Comment:
inconsequential, but i'd rename these to be verbs : `SubscribeToDynamoEvents()` and `UnsubscribeFromDynamoEvents()`, while more verbose, a bit clearer